src/status_im/ui/screens/group/views.cljs

```diff
@@ -214,9 +215,9 @@
                       :on-press            #(re-frame/dispatch [:group-chats.ui/add-members-pressed])}
           (i18n/label :t/add)]}]])))
 
-(views/defview edit-group-chat-name []
-  (views/letsubs [{:keys [name chat-id]} [:chats/current-chat]
-                  new-group-chat-name (reagent/atom nil)]
+(defn edit-group-chat-name []
+  (let [{:keys [name chat-id]} (rf/sub [:chats/current-chat])
+        new-group-chat-name    (reagent/atom nil)]
```

Comment:
atom creation should be outside render function

Reply:
Done